k-distribution/tests/regression-new/bisimulation/examples/program-equivalence/imp-lambda/simple-spec.k

```diff
@@ -43,7 +43,7 @@ claim
 </lambdaT>
 requires N1 ==Int N2
  andBool M1 ==Int M2
- andBool ?_Ks1 ==K ?_Ks2
+ andBool Ks1 ==K Ks2
```

Comment:
`andBool _Ks1 ==K _Ks2`
Don't forget about underscore, otherwise you will get warnings.